app/components/accordion_component.html.erb

```diff
@@ -3,7 +3,7 @@
     <button
       type="button"
       class="usa-accordion__button"
-      aria-expanded="false"
+      aria-expanded=<%= expanded? ? 'true' : 'false' %>
```

Comment:
1. Can we keep the double-quotes? I've historically found quotes in HTML to be optional in the same way that semicolons are "optional" in JavaScript and often cause unpredictable results.
2. Less strongly held, but could also toss a `to_s` to simplify the ternary?

```suggestion
      aria-expanded="<%= expanded?.to_s %>"
```

Reply:
if we do `expanded?.to_s` we should update the getter method to boolean cast for safety
```diff
 def expanded?
-  @expanded
+  !!@expanded
end
```

Reply:
re:chaining a predicate, we could do `String()`
```suggestion
      aria-expanded="<%= String(expanded?) %>"
```